src/store.c

```diff
@@ -173,18 +233,20 @@ struct rill_store *rill_store_open(const char *file)
         goto fail_mmap;
     }
 
-    store->head = store->vma;
-    store->vals = (void *) ((uintptr_t) store->vma + store->head->vals_off);
-    store->data = (void *) ((uintptr_t) store->vma + store->head->data_off);
-    store->index = (void *) ((uintptr_t) store->vma + store->head->index_off);
-    store->end = (void *) ((uintptr_t) store->vma + store->vma_len);
+    store->head    = store->vma;
+    store->index   = (void *) ((uintptr_t) store->vma + store->head->index_off);
+    store->index_b = (void *) ((uintptr_t) store->vma + store->head->index_b_off);
+    store->data    = (void *) ((uintptr_t) store->vma + store->head->data_off);
+    store->data_b  = (void *) ((uintptr_t) store->vma + store->head->data_b_off);
+    store->end     = (void *) ((uintptr_t) store->vma + store->vma_len);
 
     if (store->head->magic != magic) {
         rill_fail("invalid magic '0x%x' for '%s'", store->head->magic, file);
         goto fail_magic;
     }
 
     if (!is_supported_version(store->head->version)) {
+        // TODO: is the 'du' wanted here?
```

Comment:
If you want to be REALLY pedantic there's a set of macros defined for the `stdint.h` types in here http://en.cppreference.com/w/c/types/integer

But for this project, I assume that `uint32_t` is `unsigned int` and `uint64_t` is `unsigned long` so yes, `%du` is fine.

Reply:
I think that comment was due to some strangeness happening on my end  at the time. Don't mean to be super pedantic. :see_no_evil: